codice/ddf
# DDF-4251 Converts defines into require in catalog-ui-search

#### What does this PR do?

Converts defines into require in catalog-ui-search.

**NOTE** The last commit can be ignore as it is purely a formatting commit.

#### Who is reviewing it? 

@Bdthomson 
@mojogitoverhere 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@rzwiefel

#### How should this be tested?

- `mvn clean install` ui directory
- ensure ui comes up without any errors
- ensure general functionality is preserved

#### Any background context you want to provide?

Due to historical reasons, there are currently three ways to define/require/import dependencies. This PR reduces that number to 2. We could migrate define/return to import/export, but the require/module.export semantics map much better which reduces risk of regressions. The eventual goal is to migrate all code to import/export and this gets us one step closer.

#### What are the relevant tickets?

[DDF-4251](https://codice.atlassian.net/browse/DDF-4251)
